.bin/format.sh

```diff
@@ -2,5 +2,6 @@
 
 (
 pushd app || exit 1
-yarn format "$@"
+files=("$@")
+yarn format "${files[@]/#app\//}" # remove "app/" from the file names
```

Comment:
this'll work for now, but I think what we really want is to resolve these to be absolute paths